### PR TITLE
Update example app to pull in the latest fhir-client, and fhir-client-cerner-additions js projects

### PR DESCRIPTION
**Pull request by kolkheang**
Summary of changes:
- Update fhir-client js with the latest minified version
- Include fhir-client-cerner-additions js project to prevent displaying incorrect patient in Chart

@kpshek 
@mjhenkes 
@zplata

### CODE REVIEW

**Comment by kpshek** [comment]
I wouldn't pull in the minified version of the JS libraries. Given that this is a tutorial, it would be helpful if developers had access to the non-minified source for learning/debugging.

File: source/example-smart-app/launch.html [hunk 1]
```diff
@@ -7,7 +7,11 @@
     <title>Example-SMART-App</title>
   </head>
   <body>
-    <script src='./lib/fhir-client.js'></script>
+    <script src='./lib/fhir-client.min.js'></script>
```
Comment (mjhenkes) [inline source/example-smart-app/launch.html]: Can you add a comment here of what version we're pulling in?

Comment (kolkheang) [inline source/example-smart-app/launch.html]: will do

**Comment by kolkheang** [comment]
Addressed comments

**Comment by koushic88** [comment]
+1

**Pull request merged**